unlock-app/src/hooks/useRsvp.ts

```diff
@@ -33,3 +49,37 @@ export const useRsvp = ({ lockAddress, network }: Options) => {
     retry: 2,
   })
 }
+
+// fetch the number of RSVPs for an event
+export const useEventRSVP = ({
+  lockAddress,
+  network,
+  eventEndDate,
+}: UseEventRSVPProps) => {
```

Comment:
We should probably check for _all_ locks for an event for completeness. So here this function would use the checkout Config and loop over each lock.

Reply:
true, good catch! will fix!